src/app/app.component.html

```diff
@@ -1,7 +1,2 @@
-<!-- Login link test -->
-<!-- Uses authenticated variable in login.service.ts -->
-<div class="container" *ngIf="service.authenticated">
-    <h2>Logged in!</h2>
-    <p>Congrats you are now logged in!</p>
-</div>
+<app-workerapp></app-workerapp>
 <router-outlet></router-outlet>
```

Comment:
You are supposed to make this a route? Not just replace the entire app with ur site?